tests/test_gen_rid.py

```diff
@@ -0,0 +1,47 @@
+from context import rid
+import unittest, os, shutil
+
+
+class TestMakeRid(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        if os.path.exists("tmp_rid"):
+            shutil.rmtree("tmp_rid")
+        os.mkdir("tmp_rid/")
+        pass
+
+    def setUp(self):
+        self.test_dir = "tmp_rid"
+        self.test_rid_out = "tmp_rid/test_out"
+        self.benchmark_json_dir = "benchmark_json"
+        self.benchmark_mol_dir = "benchmark_mol"
+        self.json_file = os.path.join(self.benchmark_json_dir, "rid.json")
+        # self.benchmark_dir = "benchmark_rid"
+    
+    def test_gen_rid(self):
+        
+        rid.gen_rid(out_dir=self.test_rid_out, mol_dir=self.benchmark_mol_dir, rid_json=self.json_file)
+        self.assertTrue(os.path.exists(self.test_rid_out))
```

Comment:
Is there any file or sub-directory in `self.test_rid_out`? If any we may also want to test the files and sub-directories are successfully generated.